openmls/src/group/mls_group/mod.rs

```diff
@@ -494,3 +494,23 @@ pub struct StagedWelcome {
     // information.
     group: StagedCoreWelcome,
 }
+
+/// A parsed, but not fully processed `Welcome` message.
+///
+/// This may be used in order to retrieve information from the `Welcome` about
+/// the ratchet tree.
+///
+/// Use `into_staged_welcome` to get the [`StagedWelcome`] on this.
```

Comment:
```suggestion
/// A `Welcome` message that has been processed but not staged yet.
///
/// This may be used in order to retrieve information from the `Welcome` about
/// the ratchet tree and PSKs.
///
/// Use `into_staged_welcome` to stage it into a [`StagedWelcome`].
```